pkg/label/label.go

```diff
@@ -2,11 +2,34 @@
 // https://github.com/giantswarm/fmt/blob/master/kubernetes/annotations_and_labels.md.
 package label
 
-const (
-	// App label is used to identify Kubernetes resources.
-	App = "app"
+import (
+	"fmt"
 
-	// AppKubernetesVersion label is used to identify the version of Kubernetes
-	// resources.
-	AppKubernetesVersion = "app.kubernetes.io/version"
+	"github.com/giantswarm/apiextensions/v2/pkg/label"
+	"k8s.io/apimachinery/pkg/labels"
+
+	"github.com/giantswarm/app-operator/v2/pkg/project"
 )
+
+func AppVersionSelector(unique bool) labels.Selector {
+	version := GetProjectVersion(unique)
+	s := fmt.Sprintf("%s=%s", label.AppOperatorVersion, version)
+
+	selector, err := labels.Parse(s)
+	if err != nil {
+		panic(fmt.Sprintf("failed to parse selector %#q with error %#q", s, err))
+	}
+
+	return selector
+}
+
+func GetProjectVersion(unique bool) string {
```

Comment:
We need to put the same app-operator version labels into configmaps. To make it easier to get this value, I'm separating this function.